src/main/kotlin/com/openlattice/shuttle/controllers/IntegrationsContoller.kt

```diff
@@ -12,8 +12,8 @@ import javax.inject.Inject
 @RestController
 @RequestMapping("/integrations")
 class IntegrationsContoller {
-    @Inject
-    private lateinit var integrations: IntegrationsService
+//    @Inject
+//    private lateinit var integrations: IntegrationsService
```

Comment:
remove before merging?

Reply:
For sure, I think this is far from ready for merge. But will do!